extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +221,33 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							let userPassword: string | undefined;
+							// Ask user to enter password if auth type is sql login and password is not saved
+							if (connectionInfo?.authenticationType === 'SqlLogin' && !connectionInfo?.password) {
+								userPassword = await vscode.window.showInputBox({
+									prompt: constants.enterPasswordPrompt,
+									placeHolder: constants.enterPasswordManually,
+									ignoreFocusOut: true,
+									password: true,
+									validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
+								});
+								if (userPassword) {
+									// if user enters password replace password placeholder with users enter password
+									connectionString = connectionString.replace(constants.passwordPlaceholder, userPassword);
+								}
+							}
+							if ((includePassword !== constants.yesString) && !userPassword && connectionInfo?.authenticationType === 'SqlLogin') {
```

Comment:
It seems clearer to have includePassword just be a boolean instead

Reply:
I'll keep it for now as it is possible that we may change the prompts, so I want it to be clear that right now it's either yes or no to include the password. Will change later if it ends up being confusing.